### ISSUE TITLE
Tables can not be filtered

### ISSUE THREAD

**Issue by labeoVlad** [Trivadis/sqldev-jdbc-proxy#5]
One of the main functionalities that I am using on daily basis is filtering tables. But it is not working currently. If we talk Snowflake we assume DWH, as a result, hundreds of tables might be expected to be shown in the Tables pane. You don't want to browse all these tables using your eyes, you want to filter them as you normally would do with Oracle tables.

On the picture below, **Apply Filter...** functionality is not available 

![image](https://user-images.githubusercontent.com/4750556/107588412-9c53b300-6bd1-11eb-9200-66a0325ed6a0.png)

**Comment by PhilippSalvisberg** [comment]
I agree, filtering is a very helpful. However, SQL Developer supports filtering only for Oracle Database and TimesTen Database connections. Technically it would work also for MySQL connections, but Oracle decided not to support it. 

From a driver point of view it is not possible to override this behavior. And we cannot mimic an Oracle JDBC driver, since this driver is part of SQL Developer and there is no mechanism foreseen to replace it.

Filtering is one of many features that are missing when you use a third party JDBC driver. It would be possible to provide filtering by a dedicated SQL Developer extensions. But this is beyond the scope of this project.

Simply put: Only features that are available via a non-proxied mySQL connection can be expected to work on a target connection of this JDBC proxy driver.
